source/material_model/rheology/strain_dependent.cc

```diff
@@ -338,6 +338,25 @@ namespace aspect
         viscous_strain_weakening_factors = Utilities::MapParsing::parse_map_to_double_array(prm.get("Prefactor strain weakening factors"),
                                            options);
 
+        options.property_name = "Prefactor strain weakening first temperature";
```

Comment:
You still need to declare these parameters in `declare_parameters`, right?

I wonder if it is possible to find more expressive parameter names. What do you think about:
- `Lower temperature for onset of strain weakening`
- `Lower temperature for maximum strain weakening`
- `Upper temperature for maximum strain weakening`
- `Upper temperature for onset of strain weakening`

Reply:
thanks, I had forgotten about the declare_parameters.